src/runtime.js

```diff
@@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installStorageService(global);
```

Comment:
Definitely. Would you be in favor of putting it into a micro-task for now and moving back once we register factories instead of instances?